src/components/navigation/header/index.tsx

```diff
@@ -119,8 +119,8 @@ const Navigation: FC<NavigationProps> = ({
         borderBottomColor="gray.200"
         borderBottomWidth="1px"
         zIndex="header"
-        position="relative"
         backgroundColor="white"
+        {...(isOpen ? { position: 'fixed', top: 0 } : { position: 'relative' })}
```

Comment:
The opened drawer is `fixed` too, changes that to have more consistent behaviour on iOS